README.md

```diff
@@ -0,0 +1,144 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [📹 Video Walkthrough](#video-walkthrough)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 HTML Capstone: Conference page <a name="about-project"></a>
+
+> This is the Capstone project of the first module of the Microverse curriculum.
+
+**HTML Capstone: Conference page** is a project to showcase my web development skills acquired in the first module.
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you improve your project description so it includes some details about your webpage i.e. your description should tell about the application itself

Reply:
Hi @tresorsawasawa👋, Thank you for the suggestion. I have updated the project description to include more details about the application itself. Please let me know if there is anything else that needs to be addressed.